src/midend/binaryAnalyses/instructionSemantics/DataFlowSemantics2.C

```diff
@@ -77,6 +77,7 @@ public:
     createOptionalMerge(const BaseSemantics::SValuePtr &other, const BaseSemantics::MergerPtr&,
                         SMTSolver*) const ROSE_OVERRIDE {
         TODO("[Robb P. Matzke 2015-08-10]");
```

Comment:
OK, I am not sure what changed, but I removed all of these returns and it still built.  I know I had errors at one point...   So, I will retract this part of the change.